docs/cli/etcd-snapshot.md

```diff
@@ -4,47 +4,88 @@ title: etcd-snapshot
 
 # k3s etcd-snapshot
 
-This page documents the management of etcd snapshots using the `k3s etcd-snapshot` CLI, as well as configuration of etcd scheduled snapshots for the `k3s server` process, and use of the `k3s server --cluster-reset` command to reset etcd cluster membership and optionally restore etcd snapshots.
+This page describes how to use the `k3s etcd-snapshot` CLI tool to manage etcd snapshots and how to restore from an etcd snapshot.
 
-## Creating Snapshots
+K3s snapshots are stored in the node file system and optionally be replicated in an S3 compatible object store for disaster recovery scenarios. Moreover, K3s snapshots can be scheduled or taken manually on-demand.  The `k3s etcd-snapshot` CLI tool supports a set of subcommands for working with both your etcd scheduled or on-demand snapshots, as well as local or S3 stored snapshots.
```

Comment:
Simplified the language here a bit. We should also be consistent in calling these etcd snapshots, not k3s snapshots.

```suggestion
K3s etcd snapshots are stored on the node file system, and may optionally be uploaded to an S3 compatible object store for disaster recovery scenarios. Snapshots can be scheduled or taken manually on-demand. The `k3s etcd-snapshot` CLI tool offers a set of subcommands that can be used to create, delete, and manage snapshots.
```

Reply:
looks good, thanks Brad